src/ast/desugar.rs

```diff
@@ -274,8 +274,10 @@ pub(crate) fn desugar_command(
             result
         }
         Command::Sort(sort, option) => vec![NCommand::Sort(sort, option)],
-        // TODO ignoring cost for now
         Command::AddRuleset(name) => vec![NCommand::AddRuleset(name)],
+        Command::CombinedRuleset(name, subrulesets) => {
+            vec![NCommand::CombinedRuleset(name, subrulesets)]
+        }
```

Comment:
Would you be able to prefix this with unstable too? That way in the Python bindings the command will be prefixed with that as well.

Reply:
Great idea